tests/io/mscw_ctm/test_additional_variables.py

```diff
@@ -80,6 +82,22 @@ def test_calc_concNtnh():
     assert (concNtnh == concNtnh_from_func).all()
 
 
+def test_calc_concpm10pm25():
+    concpm10 = create_fake_MSCWCtm_data()
+    concpm25 = create_fake_MSCWCtm_data()
+
+    ratpm10pm25_from_func = calc_ratpm10pm25(concpm10, concpm25)
+    assert ratpm10pm25_from_func.attrs["units"] == "1"
```

Comment:
Some more extensive tests would be nice, eg on the shape of the output array and maybe even the calculated values.